blog/_posts/2017-07-17-ifPairElseUnknown.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Team ifPairElseUnknown
+layout: post
+created_at: Monday July 17 2017
+permalink: blog/2017-07-17-ifPairElseUnknown
+current: blog
+author: ifPairElseUnknown
+twitter: IfPairElse  ‏
+categories:
+- student-posts
+---
+
+# __Hello World!__
+
+![Team Banner](/img/blog/2017/ifPairElseUnknown_v5.png)
```

Comment:
please add image credits to all the images you've used in the post. “Image credits” means either a source for the images/where you got them from, or who took the photo — depending on what type of image it is. you can find more about this [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images). Please also take a look at past blog posts to see the styling that's been applied to the credits :)